### ISSUE TITLE
0.6.0 rpm seems to have wrong kong.core.reports

### ISSUE THREAD

**Issue by worktick** [Mashape/kong#905]
Hey guys

Just pulled this down and installed on a vanilla CentOS6.7 machine and received a warning about "reports.enable" in core/kong.lua (line: 135). I cracked open kong/core/reports.lua and the file doesn't match what's in the master or next branches - it's missing the .enable function.

Could you repackage pleasums?

**Comment by thefosk** [comment]
@worktick I have just pulled this file https://github.com/Mashape/kong/releases/download/0.6.0/kong-0.6.0.el6.noarch.rpm and after installing it I am seeing the `enable` function being returned by `/kong/core/reports.lua`. This are the ending lines:

```lua
return {
  init_worker = function()
    if enabled then
      cache.rawset(cache.requests_key(), 0, 0) -- Initializing the counter
      create_timer(INTERVAL, send_ping)
    end
  end,
  log = function()
    if enabled then
      cache.incr(cache.requests_key(), 1)
    end
  end,
  enable = function()
    enabled = true
  end
}
```

**Comment by thibaultCha** [comment]
@worktick Would you mind showing us the stacktrace (if any) you see in the logs? I assume there is one from your original message?

**Comment by worktick** [comment]
Hey guys, thanks for looking into this - I re-ran the installation on a clean CentOS box and then reveiwed the installed files from an RPM I downloaded before reporting the issue. It all worked as expected and the file was as expected too.

Thanks again for looking into this but I think I wasted your time with this issue, the actual issue was seen on a colleagues machine and I just assumed he'd pulled the same RPM as me - I'm not sure why the file was missing the "enable" function but it's all resolved for us now.
